content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

```diff
@@ -38,6 +38,8 @@ You can set commit permissions when you first create a pull request from a fork.
 1. In the list of pull requests, navigate to the pull request that you'd like to allow commits on.
 {% data reusables.repositories.allow-maintainers-user-forks %}
 
+![Screenshot of a Pull request. On the right bottom side, a blue, clicked checkbox, labelled "Allow edits and access to secrets by maintainers".](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```

Comment:
The ! sign must be aligned with the first word in the bullet point (which is "In...the list", so that the ALT text and the screenshot are at the same level as the step or bullet point they relate to.
Also, removing the capitalization of "Pull request" and using US spelling.
Finally, added `is highlighted with an orange outline.` to be consistent with the content model we use.

```suggestion
   ![Screenshot of a pull request. On the right bottom side, a blue, clicked checkbox, labeled "Allow edits and access to secrets by maintainers" is highlighted with an orange outline.](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```